libs/components/forms/testing/src/input-box/fixtures/input-box-harness-test.component.html

```diff
@@ -0,0 +1,30 @@
+<form novalidate [formGroup]="myForm">
+  <div class="sky-form-group">
+    <sky-input-box [attr.data-sky-id]="'my_input_box_first_name'">
```

Comment:
It might be good to use hyphens instead of underscores for the value here. I'd want to see what's more common with our consumers and then establish that as a pattern.

Reply:
Good call. Done.